src/Windows/Avalonia.Win32.Interop/Avalonia.Win32.Interop.csproj

```diff
@@ -1,9 +1,9 @@
-<Project Sdk="MSBuild.Sdk.Extras">
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

Comment:
I am not sure, but ".WindowsDesktop" part is optional only with "net5.0-window" target, and before that we still need it, right?

Reply:
Yes, I think so.